CHANGELOG.md

```diff
@@ -306,6 +306,8 @@ Shipping methods can be removed by the user after it has been assigned to a chec
 - Add missing descriptions to payment module - #13546 by @devilsautumn
 - Fix `NOTIFY_USER` allow to create webhook with only one event - #13584 by @Air-t
 - Add Index for 'Created' field of the Order Model - #13682 by @ritanjandawn
+- Fix transaction create mutation's psp_reference max length issue - #12696 by @ssuraliya
```

Comment:
We would have to move this entry to the newest version section - 3.18.

Reply:
We already moved forward with releases so when you fix this please move this to 3.19 (or just the newest unreleased version in the CHANGELOG.md)